copier/main.py

```diff
@@ -406,6 +403,7 @@ def answers(self) -> AnswersMap:
                 )
             except KeyboardInterrupt as err:
                 raise CopierAnswersInterrupt(result, question, self.template) from err
+
```

Comment:
There are people that add spaces, others remove them... let's keep only meaningful changes please 😊 

```suggestion
```